tests/conftest.py

```diff
@@ -39,6 +38,26 @@ def test_characterization_supply_curve():
     return char_csv
 
 
+@pytest.fixture
+def test_map_supply_curve_solar():

```

Comment:
Whoa I have not seen anything like this before. Is `pytest` smart enough to know these are fixtures, even though the start with `test_`? You learn something new every day...